helpers/utils.js

```diff
@@ -25,3 +25,59 @@ export const QUERY_ERROR_REGEX = /^[\+&=#@;\,\w\\'áàâäãåçéèêëíìîï
  * - minimum 1 characters and maximum 100 characters
  */
 export const QUERY_CODE_REGEX = /^[a-z0-9_-]{10,100}$/i;
+
+/**
+ * List of groups of scopes to help in login
+ */
+export const GROUPS_FAMILY_SCOPES = {
```

Comment:
« GROUPS » is quite redoudant with « FAMILY » . I rather use a simple variable name : SCOPES_GROUPS